specification/compute/resource-manager/readme.md

```diff
@@ -61,6 +61,14 @@ directive:
       - $.definitions.Snapshot.properties
     suppress:
       - BodyTopLevelProperties
+  - where:
+      - $.definitions.RestorePointCreate.properties
```

Comment:
Why is this needed?

Reply:
I requested and was granted approval for this suppression. Without it, the automated Linter fails.
Person Requesting Approval: roip@microsoft.com Submission Time: 3/10/2021 2:52:33 AM Link to PR: https://github.com/Azure/azure-rest-api-specs/pull/13219 Summary of the violations: 1.R2062 - XmsResourceInPutResponse 2. R3006 BodyTopLevelProperties Justification for supression: Regarding R2062: My resource is a proxy-only resource (not an ARM top-level resource), so I don't believe this rule applies.

Regarding R3006: My service has been already using this ‘RestorePoint’ model definition with these properties as top level properties for many years now. I cannot change these properties to suddenly now be nested under a top level property.

I will add a comment